docusaurus.config.js

```diff
@@ -56,11 +56,6 @@ module.exports = {
       additionalLanguages: ["ruby", "php", "java", "groovy", "csharp", "rust", "kotlin"],
     },
   },
-  scripts: [
```

Comment:
You want to delete the script itself too, no?

Reply:
oops, missed that. Removed it now.